plugins/harness-ci-cd/src/components/ExecutionList/ExecutionList.tsx

```diff
@@ -745,83 +744,88 @@ function ExecutionList() {
 
   function getSecureHarnessKey(key: string): string | undefined {
     try {
-      const token = JSON.parse(decodeURI(atob(localStorage.getItem(key) || '')));
+      const token = JSON.parse(
+        decodeURI(atob(localStorage.getItem(key) || '')),
+      );
       return token ? `Bearer ${token}` : '';
     } catch (err) {
-      console.error('Failed to read Harness tokens', err);
+      // eslint-disable-next-line no-console
+      console.log('Failed to read Harness tokens', err);
       return undefined;
     }
   }
-  
+
   async function fetchLicenseWithAuth(): Promise<Response> {
     const token = getSecureHarnessKey('harnessBearerToken');
     const value = token ? `Bearer ${token}` : '';
-  
+
     const headers = new Headers({
-      'Authorization': value,
+      Authorization: value,
     });
     const response = await fetch(
-      `${await backendBaseUrl}/harness/${env}/gateway/ng/api/licenses/account?routingId=${accountId}&accountIdentifier=${accountId}`, 
+      `${await backendBaseUrl}/harness/${env}/gateway/ng/api/licenses/account?routingId=${accountId}&accountIdentifier=${accountId}`,
       {
-        headers
-      }
+        headers,
+      },
     );
-  
+
     return response;
   }
-  
+
   async function getLicense() {
     const response = await fetchLicenseWithAuth();
-  
+
     if (response.status === 200) {
       const data = await response.json();
       if (data?.data?.allModuleLicenses?.CD?.length === 0) {
         setLicenses('ci');
       }
     }
   }
-  
+
   async function getPipeLineByService() {
     const list = serviceId;
     const service1 = list?.split(',').map(item => item.trim()) || '';
-  
+
     const token = getSecureHarnessKey('harnessBearerToken');
     const value = token ? `Bearer ${token}` : '';
```

Comment:
`getSecureHarnessKey` will return in format `Bearer ${token}`. We are adding Bearer again.